framework/console/controllers/AssetController.php

```diff
@@ -544,10 +549,10 @@ protected function compressCssFiles($inputFiles, $outputFile)
         if (is_string($this->cssCompressor)) {
             $tmpFile = $outputFile . '.tmp';
             $this->combineCssFiles($inputFiles, $tmpFile);
-            $this->stdout(shell_exec(strtr(Yii::getAlias($this->cssCompressor), [
+            $this->stdout(shell_exec(strtr($this->cssCompressor, ArrayHelper::merge($this->compressorAdditionalParameters, [
```

Comment:
What's reason to using ArrayHelper::merge() instead `[...] + [...]` or `array_merge()`?